Utilities/run-full-tests.sh

```diff
@@ -69,7 +69,7 @@ try() {
 try "spm.debug.build" $swift build -c debug $spm_flags --build-path "$build_dir/spm.debug"
 try "spm.release.build" $swift build -c release $spm_flags --build-path "$build_dir/spm.release"
 if [ "$(uname)" = "Linux" ]; then
-    try "spm.release.dword.build" $swift build -c release $spm_flags -Xcc -mcx16 -Xswiftc -DENABLE_DOUBLEWIDE_ATOMICS --build-path "$build_dir/spm.release.dword"
+    try "spm.release.dword.build" $swift build -c release $spm_flags --build-path "$build_dir/spm.release.dword"
 fi
```

Comment:
Nit: we should just remove this entire `if` altogether -- this configuration is now the default, so it is rolled into the default Linux case.